modules/ROOT/pages/sso-prerequisites-about.adoc

```diff
@@ -35,5 +35,4 @@ https://anypoint.mulesoft.com/accounts/login/{your_org_domain}
 * OpenID Connect does not support role mapping.
 * If a SAML user belongs to certain groups, Anypoint Platform does not automatically grant equivalent roles in the organization.
 * Anypoint Platform does not generate the SAML assertion for single sign on. Your IdP generates the Sign On URL that you configure.
```

Comment:
```suggestion
* Anypoint Platform does not generate the SAML assertion for single sign on. 
Your IdP generates the sign-on URL that you configure.
```

Reply:
GLOBALLY break steps or other list items after the first sentence or fragment, to increase scannability.